docs/TempUG.md

```diff
@@ -0,0 +1,331 @@
+3. Task Lists: keep track of tasks to be done for lessons
+
+With a customised student list and schedule in TutorMate just for you, organising lessons and managing students
+will be the least of your worries.
+
+This project is based on the [AddressBook-Level3 project](https://se-education.org).
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Quick start
+
+1. Ensure you have Java 11 or above installed in your Computer. You can check by opening a command terminal and typing `java -version`.
+
+1. Download the latest `tutormate.jar` from [here](https://github.com/AY2324S1-CS2103T-T11-3/tp/releases).
+
+1. Copy the file to the folder you want to use as the _home folder_ for your TutorMate.
+
+1. Open a command terminal, navigate into the folder you put the jar file in using the change directory command `cd`, and use the `java -jar tutormate.jar` command to run the application.<br>
+   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+   ![Ui](images/about.png)
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing `help` and pressing Enter will open the help window.<br>
+   Some example commands you can try:
+
+    * `list students` : Lists all students with their name.
+
+    * In list `STUDENTS`:
+
+        * `add -name Leah` : Adds a student named "Leah" to the application.
+
+        * `show 3` : Shows the details of the person with the index 3 in TutorMate.
+
+    * `exit` : Exits the app.
+
+1. Refer to the [Features](#features) below for details of each command.
+2. Refer to the [Command Summary](#command-summary) below for the summary of all commands.
+2. Refer to the [Glossary](#glossary) below for definitions of glossary terms.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Important notes
+
+### Basic usage
+* The app is split into 3 states: student list, schedule list and task list. Each corresponds to the main features of TutorMate.
+* Each state has its associated features, while certain features work with all states but has differing functionalities.
+* The student list handles student details management, schedule list handles lessons and scheduling, task list handles tasks.
+* The GUI has several main components (see GUI image below):
+  * The command box is for users to enter and execute commands.
+  * The response box is to display responses for command execution, to indicate success or errors.
+  * The left side has the list panel, which shows different list types (student, schedule, tasks).
+  * The right side has the details panel, which shows details of any specific item in the list.
+![Ui](images/Ui.png)
+
+### Terminologies / Symbols
+* Flag: denoted with a dash before the flag name e.g. -name.
```

Comment:
should we add one for parameters also?
e.g.
Parameters: Text that follows a flag (or something)